### ISSUE TITLE
Open files tabs (add: auto-sort tabs feature)

### ISSUE THREAD

**Issue by tiomanisland** [notepad-plus-plus/notepad-plus-plus#8147]
### Description of the Issue
I would like to see auto-sort-tabs feature added to Notepad++

### Steps to Reproduce the Issue

1.  Open test.php
2.  Open config.php
3.  Open hello.php

### Expected Behavior

I would expect files to open in tabs sorted from A-Z, means that tabs would look like:

1. config.php
2. hello.php
3. test.php

### Actual Behavior

File tabs are not sorted. Notepad++ is just adding files after each other. There could be settings that would keep current behaviour (not sorted) and X auto-sort tabs (in real time).

### Debug Information

When you have 20 files opened and you need to fix something in config.php your eyes must go left-right-left-right to find that file.

**Comment by sasumner** [comment]
See https://community.notepad-plus-plus.org/topic/19084/

Suggest *NavigateTo* plugin -- why visually scan  when you could just start typing "c...o...n..."? By then you've probably found your file.

**Comment by tiomanisland** [comment]
This is not what I said. I was talking about real time auto-sort feature, without a need to select option from menu or click or type.

**Comment by sasumner** [comment]
>This is not what I said.

Clearly.

Comments here are for offering workarounds to things that (probably) won't be implemented.

**Comment by tiomanisland** [comment]
I get it. I will try what you said. Do you know any editor with such
feature?

**Comment by sasumner** [comment]
>Do you know any editor with such feature?

I have never seen such a feature after using GUI text editors for 25 years.

**Comment by tiomanisland** [comment]
I saw it but only in one editor it was called something like jdedit
or j**editor made from a guy in poland or czech
There is also VS plugin they say but i dont use VS
Nice feature, I am surprised others don't add such a simple feature to
implement.

**Comment by ArkadiuszMichalski** [comment]
Another ask:
https://community.notepad-plus-plus.org/topic/20434/auto-sort-tabs-real-time

**Comment by sasumner** [comment]
>Another ask

The two Community postings are by the same individual.
This issue was opened shortly after the earlier Community posting was made.
Thus I deduce there is exactly ONE person requesting this functionality.

**Comment by ArkadiuszMichalski** [comment]
Looks like someone has sclerosis or really wants this feature. This can't be done with PythonScript? Take all tabs, map to table, sort table according to the file name, move tab one by one according to the sorted idx?

**Comment by freezer2022** [comment]
Yeah, also, instead of writing a comment in the current thread, he also 2 years later smuggled a duplicate of the current thread, in a camouflaged / altered title :
- https://github.com/notepad-plus-plus/notepad-plus-plus/issues/11400
